frontend/src/components/DropToStartField.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+const isActive = ref(false)
+function setActive() {
+  isActive.value = true
+}
+function setInactive() {
+  isActive.value = false
+}
+</script>
+<template>
+  <TheSlogan />
+  <div class="container">
+    <div class="row justify-content-md-center">
+      <div
+        class="card border-3 border-3 rounded-3 drop"
+        :data-active="isActive"
+        @dragenter.prevent="setActive"
+        @dragover.prevent="setActive"
+        @dragleave.prevent="setInactive"
+        :class="{ 'bg-light': isActive }"
+      >
+        <div class="card-body d-flex justify-content-center align-items-center">
+          <h3 class="card-title">Drop File to Start!</h3>
+        </div>
+      </div>
+    </div>
+    <div class="row justify-content-md-center text-secondary">
+      Agree to our Terms of Service and Privacy & Cookie Statement.
```

Comment:
We need add links to `Terms of Service` and `Privacy & Cookie Statement`. But I don't know what pages I should link to.

Reply:
You can create two static components in the frontend with a lorem-ipsum and we'll have Stephane provide the actual content